wordpress-mobile/WordPress-iOS
# Issue/10319 site creation verticals svc integration

### Description
Fixes #10319.

To test:

- Checkout the branch and confirm that tests pass.
- Step through the site creation flow as shown below and verify that the Verticals service responds to your query.
- Alternately toggle network connectivity and validate the error state is shown as expected.

Update release notes:


### Visual QA

![10319_pr_03](https://user-images.githubusercontent.com/221062/50938596-bc7e7a00-142d-11e9-9bfc-45a689de63b1.gif)

The code can be reviewed as-is. That said, I have affixed the `[Status] Needs Design Review` and `[Status] DO NOT MERGE` labels to the PR to reflect the fact that I'd like to obtain Visual QA input from @SylvesterWilmott regarding the fidelity of the error states. To assist with that validation, I've provided the following screenshots across a range of device sizes.

_iPhone SE : No connection_

<img width="504" alt="10319_iphonese_noconnection" src="https://user-images.githubusercontent.com/221062/50938486-4da12100-142d-11e9-9019-ade871058c41.png">

_iPad (Portrait) : Server error_

<img width="878" alt="10319_ipad_portrait_servererror" src="https://user-images.githubusercontent.com/221062/50938489-52fe6b80-142d-11e9-97fc-0a4236b4af4f.png">

_iPad (Landscape) : Server error_

<img width="1109" alt="10319_ipad_landscape_servererror" src="https://user-images.githubusercontent.com/221062/50938508-5db90080-142d-11e9-9b5f-231418813e06.png">
